src/main/java/net/neoforged/neoforge/registries/DataPackRegistryEvent.java

```diff
@@ -74,6 +75,29 @@ public <T> void dataPackRegistry(ResourceKey<Registry<T>> registryKey, Codec<T>
             this.registryDataList.add(new DataPackRegistryData<>(new RegistryDataLoader.RegistryData<>(registryKey, codec, false), networkCodec));
         }
 
+        /**
+         * Registers the registry key as a datapack registry with a given {@link RegistryBuilder} to configure, which will cause data to be loaded from
```

Comment:
```suggestion
         * Registers the registry key as a datapack registry with a {@link RegistryBuilder} configurator, which will cause data to be loaded from
```

Reply:
I have added a new commit to resolve this